build.gradle.kts

```diff
@@ -67,9 +75,23 @@ tasks.named("assemble").configure {
 tasks.withType<JavaExec>().all {
 	jvmArgs("--enable-preview")
 	// Override the library path for tasks like run and test so they can find the libraries.
-	systemProperty("stacker.library.path", file("src/main/dist/native").absolutePath)
+	systemProperty("stacker.library.path", file("native").absolutePath)
 }
 
 tasks.withType<JavaCompile>().all {
 	options.compilerArgs!! += "--enable-preview"
 }
+
+val requireNativeTask by tasks.register("requireNativeDirectory") {
+	inputs.dir("native")
+
+	val file = file("native")
+	check(file.exists() && file.length() > 0L) {
+		"'native' directory does not exist or is empty. This must be populated with libgit libraries.\n" +
+			"If developing locally, please run .github/workflows/build-deps.sh"
+	}
```

Comment:
This should be in a `doFirst` or `doLast` block. Right now it runs during configuration.

I think you can also do `check(!files(native).isEmpty)` because you also want to guard against the case where there's empty directories within the `native/` folder.

Reply:
That files API doesn't work like that. I went with `file("native").walk().any { it.isFile }`